src/components/NavBar.js

```diff
@@ -0,0 +1,15 @@
+import { Link } from 'react-router-dom';
+import './NavBar.css';
+
+const NavBar = () => (
+  <nav className="navbar">
+    <span className="title">Math Magicians</span>
+    <div className="links">
+      <Link to="/home">Home</Link>
```

Comment:
- [OPTIONAL ] Let's make our Home page to be a default page the one that will be showing up when users will open your app. To make it, you only need to remove the path eg ` <Link to="/">Home</Link>` this will make your home page to be the default one 👍🏻